scripts/autoexec.lua

```diff
@@ -50,7 +52,7 @@ function enableModelAnimReplaceFlags(m_id, copy_meshes_back)
     end;
 end;
 
-if (getLevelVersion() < TR_II) then
+if (ver == TR_I) or (ver == TR_I_DEMO) or (ver == TR_I_UB) then
```

Comment:
ok, why not (ver < TR_II) - only one comparison, can not forget DEMO, GOLD, OTHER_SPECIAL editions.